### DISCUSSION TITLE
LUKS TPM2/Pin Support

### DISCUSSION THREAD

**Discussion by 3mpSh4ner** [rhinstaller/anaconda#4393] [Ideas]
Since F37+ uses systemd >251, it would be good if the LUKS Setup allows for
using TPM2 + PIN (and possibly other keydevices like FIDO2, PKCS11).

Shouldn't be an important feature as you can switch the key after the setup, but definitely nice to have.

**Comment by jkonecny12** [comment]
Hi, we had already a few requests for handling TPM support. Unfortunately, we are not able to find capacity for adding that. In other word we would like to add the support in the future but there is no ETA right now.

**Comment by h908714124** [comment]
Imho signed systemd-boot is a prerequisite for this. Which seems to be sidelined, see https://pagure.io/releng/issue/10765
